features/FEATURE_BLE/ble/generic/GenericGap.h

```diff
@@ -620,6 +620,17 @@ class GenericGap :
         phy_t rx_phy
     );
 
+    virtual void on_att_mtu_changed(
+        Handle_t connection_handle,
```

Comment:
please use `ble::connection_handle_t`.

Reply:
just copied existing types in the file, will change